apps/qubit/modules/settings/actions/globalAction.class.php

```diff
@@ -273,6 +275,16 @@ protected function updateGlobalSettings()
       $setting->save();
     }
 
+    // Escape queries, add setting if it's not already created (to avoid adding it in a migration)
+    if (null === $setting = QubitSetting::getByName('escape_queries'))
+    {
+      $setting = QubitSetting::createNewSetting('escape_queries', null);
```

Comment:
Hi @MikeFE, the default value is set in the form and added to the setting after this condition. We have also decided to leave it empty by default to avoid unexpected changes in upgrades.